### PR TITLE
Stories display

### PR DESCRIPTION
**Pull request by teehamaral**
This PR is about the issue #288

### CODE REVIEW

**Comment by norkans7** [comment]
@teehamaral @nelsonlapf I agree with the changes on the homepage but not with those on the stories page,

The stories page should remain as it is since the non featured stories are just listed under the section of 'Archived Stories'.

**Comment by teehamaral** [comment]
Ok @norkans7, I'll revert the changes on the stories page

File: ureport/public/tests.py [hunk 1]
```diff
@@ -335,26 +334,17 @@ def test_index(self, mock_cache_get):
 
         self.assertTrue(response.context['stories'])
         self.assertTrue(story1 in response.context['stories'])
-        self.assertFalse(response.context['other_stories'])
```
Comment (norkans7) [inline ureport/public/tests.py]: I think you need to keep these tests as they were, actually this was testing stories from a different org are filtered out correctly, I would revert these tests and only adjust those that relate to 'other_stories' key

File: ureport/public/tests.py [hunk 2]
```diff
@@ -455,6 +440,12 @@ def test_index(self, mock_cache_get):
         response = self.client.get(home_url, SERVER_NAME='nigeria.ureport.io')
         self.assertTrue('<div>INCLUDE MY CUSTOM HTML</div>' in response.content)
 
+    def test_additional_menu(self):
```
Comment (norkans7) [inline ureport/public/tests.py]: +1 for more tests

**Comment by norkans7** [comment]
Looks good now, thanks @teehamaral

**Pull request merged**